dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryCacheImpl.java

```diff
@@ -45,11 +45,10 @@ protected Category get(String id) throws DotDataException {
 
 	@SuppressWarnings("unchecked")
 	@Override
-	protected List<String> getChildren(Categorizable parent) throws DotDataException {
+	protected List<Category> getChildren(Categorizable parent) throws DotDataException {
 		List<String> childrenIds = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "childrenIds" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)